src/entity.js

```diff
@@ -570,6 +574,17 @@ Entity.configure = function(options) {
  *   },
  * }
  *
+ * To use an in-memory, testing-oriented table, replace the `account` key with
+ * `inMemory`, and do not supply credentials:
+ * {
+ *   inMemory: true,
```

Comment:
I had been expecting to not use a configuration file for tests, but now that you mention it it might be nice if we can run the same tests in both the real-azure and fake.